apps/email/js/cards/fld/account_item.html

```diff
@@ -1,4 +1,4 @@
 <a class="fld-account-item">
   <span class="selected-indicator"></span>
-  <span dir="auto" class="fld-account-name"></span>
-</a>
\ No newline at end of file
+  <span class="fld-account-name"></span>
```

Comment:
dir="auto" was our previous guidance. I'm fine changing that though if the common approach for apps has worked out something better, but to me that implies changing the other uses of dir="auto" in email to all use this same pattern, vs just this one element?